test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-fatalerrors = false
+fatalerrors = true
```

Comment:
Probably better keep `false` here, as it allows running all tests to see all errors, and only fail at the end.